src/tests/ftest/util/daos_utils.py

```diff
@@ -452,57 +452,37 @@ def container_get_attr(self, pool, cont, attr, sys_name=None):
                 Defaults to None.
 
         Returns:
-            dict: Dictionary that stores the attribute and value in "attr" and
-                "value" key.
+            CmdResult: Object that contains exit status, stdout, and other
```

Comment:
But then don't you need to fix users of container_get_attr(), like in query_attribute.py ??

Reply:
> 
> 
> But then don't you need to fix users of container_get_attr(), like in query_attribute.py ??

Hmm, yes. Looks like. We should also fix the tags on that test so that we can use Feature: container to test this sort of thing better.

Reply:
The `_get_json_result()` method does return a dictionary (https://github.com/daos-stack/daos/blob/master/src/tests/ftest/util/command_utils.py#L631) per https://docs.python.org/3/library/json.html#json.loads:
```suggestion
            dict: the dmg json command output converted to a python dictionary
```

Reply:
Updated query_attribute.py as well.